docs/management/connectors/action-types/email.asciidoc

```diff
@@ -95,51 +188,48 @@ image::management/connectors/images/email-params-test.png[Email params test]
 
-To, CC, BCC::    Each item is a list of addresses. Addresses can be specified in `user@host-name` format, or in `name <user@host-name>` format. One of To, CC, or BCC must contain an entry.
-Subject::       The subject line of the email.
-Message::       The message text of the email. Markdown format is supported.
+To, CC, BCC::
+Each item is a list of addresses. Addresses can be specified in `user@host-name` 
+format, or in `name <user@host-name>` format. One of To, CC, or BCC must contain 
+an entry.
+
+Subject::
+The subject line of the email.
+
+Message::
+The message text of the email. Markdown format is supported.
 
 [float]
 [[configuring-email]]
 ==== Configuring email accounts for well-known services
 
-The email connector can send email using many popular SMTP email services and the Microsoft Exchange Graph API.
+The email connector can send email using many popular SMTP email services and 
+the Microsoft Exchange Graph API.
 
-For more information about configuring the email connector to work with different email systems, refer to:
+For more information about configuring the email connector to work with 
+different email systems, refer to:
 
 * <<elasticcloud>>
 * <<gmail>>
 * <<outlook>>
 * <<amazon-ses>>
 * <<exchange>>
 
-For other email servers, you can check the list of well-known services that Nodemailer supports in the JSON file https://github.com/nodemailer/nodemailer/blob/master/lib/well-known/services.json[well-known/services.json].  The properties of the objects in those files &mdash; `host`, `port`, and `secure` &mdash; correspond to the same email connector configuration properties.  A missing `secure` property in the "well-known/services.json" file is considered `false`.  Typically, `port: 465` uses `secure: true`, and `port: 25` and `port: 587` use `secure: false`.
+For other email servers, you can check the list of well-known services that 
+Nodemailer supports in the JSON file 
+https://github.com/nodemailer/nodemailer/blob/master/lib/well-known/services.json[well-known/services.json]. 
+The properties of the objects in those files &mdash; `host`, `port`, and 
+`secure` &mdash; correspond to the same email connector configuration 
+properties. A missing `secure` property in the "well-known/services.json" file 
+is considered `false`.  Typically, `port: 465` uses `secure: true`, and 
+`port: 25` and `port: 587` use `secure: false`.
 
 [float]
 [[elasticcloud]]
 ==== Sending email from Elastic Cloud
 
-IMPORTANT: These instructions require you to link:{cloud}/ec-watcher.html#ec-watcher-whitelist[allowlist] the email addresses that notifications get sent.
-
-Use the following connector settings to send email from Elastic Cloud: 
-
-Sender::
-`noreply@watcheralert.found.io`
-
-Service::
-`elastic_cloud`
-
-Host::
-`dockerhost`
-
-Port::
-`10025`
-
-Secure::
-Toggle off
-
-Authentication::
-Toggle off
+Use the preconfigured email connector (`Elastic-Cloud-SMTP`) to send emails from 
```

Comment:
I assume it's obvious that this only works for cloud deployments, and not on-prem deployments.

I think it may be accidental (sort order) if it's true that this is the default option for new email connectors.  I don't think we want to doc that.

Reply:
@pmuellr I removed the reference to the default option.